src/Model.js

```diff
@@ -124,8 +124,8 @@ const Model = class Model {
         return this.session.getDataForModel(this.modelName);
     }
 
-    static markAccessed() {
-        this.session.markAccessed(this);
+    static markAccessed(id) {
+        this.session.markAccessed(this.modelName, id);
```

Comment:
It looks like this was actually broken? `markAccessed` expects a `modelName` but this was passing in the whole model.